spec/pane-axis-element-spec.coffee

```diff
@@ -0,0 +1,46 @@
+PaneAxisElement = require '../src/pane-axis-element'
+PaneAxis = require '../src/pane-axis.coffee'
+
+describe "PaneResizeHandleElement", ->
```

Comment:
Why do we have `PaneResizeHandleElement` both here and in another file? Could both be integration tested in a `pane-container-element-spec`? I realize the specs in this area are kind of a mess because they've never been ported away from deprecated SpacePen.